src/quest/QuestObjectiveModel.h

```diff
@@ -13,17 +15,17 @@ class QuestObjectiveModel : CYIAbstractDataModel
     QuestObjectiveModel(CYIString name);
     ~QuestObjectiveModel();
 
-    void SetUnresolvedText(CYIString text);
-    void AddResolutionText(CYIString text);
+    void AddRowsToMatchIndex(YI_INT32 index);
```

Comment:
If you think these methods don't need to have public access, you can move them into the private section.